luxonis_ml/data/datasets/annotation.py

```
@@ -528,6 +533,7 @@ class DatasetRecord(BaseModelExtraForbid):
     files: dict[str, FilePath]
     annotation: Detection | None = None
     task_name: str = ""
+    sample_metadata: Params = {}
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)